src/Firebase/Auth/LinkedProviderData.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kreait\Firebase\Auth;
+
+use Kreait\Firebase\Value\Provider;
+
+class LinkedProviderData implements \JsonSerializable
```

Comment:
Please remove this class and let the methods return "just" `UserRecord`s via the `->getUser()` method (as talked about in https://github.com/kreait/firebase-php/pull/320#issuecomment-532771266)